portfolio/src/main/java/com/google/sps/servlets/MyServlet.java

```diff
@@ -0,0 +1,49 @@
+package com.google.sps.servlets;
+
+// import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.*;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/information")
+public final class MyServlet extends HttpServlet {
```

Comment:
Minor suggestion: It may be easier to follow the code if the endpoint URL and the class name are related. E.g., if the endpoint is `/information`, the class name could be `InformationServlet`. That way there are less names to keep track of, and a code reader wouldn't have to wonder where in the code to find the class that's serving `/information`.